### ISSUE TITLE
Download dataset issues

### ISSUE THREAD

**Issue by Mkit-Itovich** [CVUsers/Smoke-Detect-by-YoloV5#13]
Hello, unfortunately your dataset has run out, could you re-upload it?

**Comment by CVUsers** [comment]
链接：https://pan.baidu.com/s/1e_1Imzvlu94-0iAstdkuHA 提取码：deep 复制这段内容后打开百度网盘手机App，操作更方便哦 欢迎分享文章~欢迎进深度学习交流群------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;&quot;Mkit-Itovich&quot;<notifications@github.com&gt;
发送时间:&nbsp;2021年3月1日(星期一) 晚上8:01
收件人:&nbsp;&quot;CVUsers/Smoke-Detect-by-YoloV5&quot;<Smoke-Detect-by-YoloV5@noreply.github.com&gt;;
抄送:&nbsp;&quot;Subscribed&quot;<subscribed@noreply.github.com&gt;;
主题:&nbsp;[CVUsers/Smoke-Detect-by-YoloV5] Download dataset issues (#13)

**Comment by Sbboss** [comment]
Hi can i get the extraction code please

**Comment by CVUsers** [comment]
wechat:zxx15277368495z
baidu pandownload:链接：https://pan.baidu.com/s/1e_1Imzvlu94-0iAstdkuHA 提取码：deep&nbsp;

---Original---
From: "Shiva ***@***.***&gt;
Date: Tue, Jan 11, 2022 20:25 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [CVUsers/Smoke-Detect-by-YoloV5] Download dataset issues (#13)




 
Hi can i get the extraction code please
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Sbboss** [comment]
Hi, actually i cant make a pan.baidu account due to country resaon. Can you please provide some work-around so that i can access the dataset. please

**Comment by CVUsers** [comment]
https://download.csdn.net/download/qq_46098574/75444687&nbsp;The audit results will be completed within 2-10 working days




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "CVUsers/Smoke-Detect-by-YoloV5"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年1月11日(星期二) 晚上9:47
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [CVUsers/Smoke-Detect-by-YoloV5] Download dataset issues (#13)





 
Hi, actually i cant make a pan.baidu account due to country resaon. Can you please provide some work-around so that i can access the dataset. please
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Sbboss** [comment]
Thanks CV but can you upload it on drive or somewhere else where i can access it faster.
I have some school project to finish. Please

**Comment by Sbboss** [comment]
> 

it is paid @CVUsers, please provide a free source

**Comment by CVUsers** [comment]
csdn平台限制，必须要钱或者csdn 的积分，不能免费



---Original---
From: "Shiva ***@***.***&gt;
Date: Wed, Jan 12, 2022 12:48 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [CVUsers/Smoke-Detect-by-YoloV5] Download dataset issues (#13)




   
it is paid @CVUsers, please provide a free source
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Sbboss** [comment]
>
yes, that is why I am asking can you please upload it on some common platform like google drive, mega, pcloud, or some other. Basically, the site that I can access in India. 
Please it will be a great help @CVUsers

**Comment by CVUsers** [comment]
我们似乎无法使用这些平台，在中国



---Original---
From: "Shiva ***@***.***&gt;
Date: Wed, Jan 12, 2022 13:05 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [CVUsers/Smoke-Detect-by-YoloV5] Download dataset issues (#13)




   
yes, that is why I am asking can you please upload it on some common platform like google drive, mega, pcloud, or some other. Basically, the site that I can access in India.
 Please it will be a great help @CVUsers
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Sbboss** [comment]
Ok can you provide me link any temporary Baidu account so that I can download the dataset? After you can revoke it.

**Comment by Sbboss** [comment]
https://www.filecloud.com/ 
Hi here is the site, you can upload it here. It is free in China and India both.

**Comment by Sbboss** [comment]
@CVUsers

**Comment by myheart111** [comment]
hi , you can use `**baidufast.com**` to download file from pan baidu or quark pan free
